src/components/enterprise-user-subsidy/enterprise-offers/EnterpriseOffersBalanceAlert.jsx

```diff
@@ -0,0 +1,57 @@
+import React, { useContext } from 'react';
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
+
+import { Container, Alert, MailtoLink } from '@edx/paragon';
+import { AppContext } from '@edx/frontend-platform/react';
+
+const EnterpriseOffersBalanceAlert = ({
+  adminText,
+  alertClassName,
+  alertVariant,
+  alertIcon,
+  alertHeading,
+  alertText,
+}) => {
+  const {
+    enterpriseConfig: { adminUsers },
+  } = useContext(AppContext);
+
+  const adminEmails = adminUsers.map(user => user.email);
+  const hasAdminEmails = adminEmails.length > 0;
+
+  const actions = hasAdminEmails ? [
+    <MailtoLink
+      to={adminEmails}
+      target="_blank"
+    >{adminText}
+    </MailtoLink>,
+  ] : [];
+
+  return (
+    <Container size="lg" className="pt-3">
+      <Alert
+        className={classNames({ [`${alertClassName}`]: hasAdminEmails })}
```

Comment:
snagged the ```[`${alertClassName}`]``` from https://www.npmjs.com/package/classnames (search for "Dynamic class names with ES2015" on the page), but not appearing the browser as i would expect so need to check up on this